tests/core_tests/block_reward.cpp

```diff
@@ -77,7 +78,7 @@ namespace
     std::vector<size_t> block_weights;
     generator.get_last_n_block_weights(block_weights, get_block_hash(blk_prev), median_block_count);
 
-    size_t median = misc_utils::median(block_weights);
+    size_t median = cryptonote::median(block_weights);
```

Comment:
Since there's `using namespace cryptonote` already there, you can ommit it here.

Reply:
IIRC this gave me a compile error. A naming conflict with the variable `size_t median`.

Reply:
Right.
Then I insist on the math namespace.